README.md

```diff
@@ -166,7 +163,7 @@ const storeEncrypted = async (payload, key) => {
   // encrypt into JWE container layout using secret key
   const jwe = await createJWE(cleartext, [dirEncrypter])
   // let IPFS store the bytes using the DAG-JOSE codec and return a CID
-  const cid = await ipfs.dag.put(jwe, { format: dagJoseIpldFormat.codec, hashAlg: 'sha2-256' })
+  const cid = await ipfs.dag.put(jwe, { format: 'dag-jose', hashAlg: 'sha2-256' })
```

Comment:
best to use use `dagJose.name` since that's the internal matching that's going to happen in js-ipfs and the user can avoid one magic string

Reply:
```suggestion
  const cid = await ipfs.dag.put(jwe, { format: dagJose.name, hashAlg: 'sha2-256' })
```